src/project-manager.ts

```diff
@@ -148,6 +142,7 @@ export class ProjectManager {
 		}
 	}
 
+	// TODO replace this with something like a WorkspaceTree class
```

Comment:
Not sure if it is worth the trouble? What advantage would it bring over the current implementation?